docs/ref/contrib/auth.txt

```diff
@@ -67,11 +67,11 @@ Fields
 
     .. attribute:: is_staff
 
-        Boolean. Designates whether this user can access the admin site.
+        Boolean. Determines whether this user can access the admin site.
 
     .. attribute:: is_active
 
-        Boolean. Designates whether this user account should be considered
+        Boolean. Indicates that this user account should be considered
         active. We recommend that you set this flag to ``False`` instead of
         deleting accounts; that way, if your applications have any foreign keys
         to users, the foreign keys won't break.
```

Comment:
```suggestion
        Boolean. Marks this user account as active. We recommend that you
        set this flag to ``False`` instead of deleting accounts; that way, if your
        applications have any foreign keys to users, the foreign keys won't
        break.
```